euth/contrib/static/js/app.js

```diff
@@ -15,3 +15,23 @@ module.exports = {
   'renderFollow': ReactFollow.renderFollow,
   'renderUserList': ReactUserList.renderUserList
 }
+
+var initilizeWidget = function (project, name, initializer) {
+  if (!initializer) {
+    initializer = name
+    name = project
+    project = 'euth'
+  }
```

Comment:
Because the api is more logical that way. From the context to the widget to the parameters.